spec/test-server.js

```diff
@@ -0,0 +1,10 @@
+const axios = require('axios');
```

Comment:
File naming: This file does not "test server", it contains a helper function that sends a get request to AWS S3.

1. I'm unsure if we need this as a helper function, since we only do it in your tests one time.
2. We can just use lines 5-7 in your actual test code instead.

Reply:
I don't know how I could test that my S3 server's photo objects are actually publicly facing and accessible without sending a GET request of some sort. Maybe I don't fully understand what you mean here.

Reply:
I just meant you can do the get request from the actual test file instead of having a helpers file. Either way is okay by me though!

Please still update the file name.